docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -49,20 +49,95 @@ refs:
 
 # Geomap
 
-Geomaps allow you to view and customize the world map using geospatial data. You can configure various overlay styles and map view settings to easily focus on the important location-based characteristics of the data.
+Geomaps allow you to view and customize the world map using geospatial data. It's the ideal visualization if you have data that includes location information and you want to see it displayed in a map. You can configure various overlay styles, like routes, as well as map view settings including adding numeric values, labels, and icons. This helps you to easily focus on the important location-based characteristics of the data.
```

Comment:
```suggestion
Geomaps allow you to view and customize the world map using geospatial data. It's the ideal visualization if you have data that includes location information and you want to see it displayed in a map. You can configure and overlay various types of layers, like markers, heatmaps, routes, and network. In this mix, you can also overlay and blend various basemaps we include or your own custom map. You can customize the map view extents and even set it to fit your data dynamically. This helps you to easily focus on the important location-based characteristics of the data.
```

Reply:
I'm wary of overloading this opening section. When you look at it in preview, it becomes a fair bit to read before the next heading. What about:
```suggestion
Geomaps allow you to view and customize the world map using geospatial data. It's the ideal visualization if you have data that includes location information and you want to see it displayed in a map.

You can configure and overlay [map layers](#types), like markers and routes, and blend included basemaps or your own custom maps. This helps you to easily focus on the important location-based characteristics of the data.
```

Reply:
That is much better. One suggestion I have is to mention "heatmaps and networks" as these are more interesting or broadly applicable use cases imo.